templates/stable-diffusion-pretraining/01_preprocess.ipynb

```diff
@@ -6,7 +6,7 @@
    "source": [
     "# Scalable data preprocessing pipeline for Stable Diffusion\n",
     "\n",
-    "<img src=\"https://anyscale-materials.s3.us-west-2.amazonaws.com/stable-diffusion/preprocessing_architecture_v4.jpeg\" width=\"900px\">\n",
+    "<img src=\"https://anyscale-materials.s3.us-west-2.amazonaws.com/stable-diffusion/preprocessing_architecture_v4.jpeg\" width=900px />\n",
```

Comment:
we have some downstream scripts that assume that these width and heights are not quoted... :( the downstream script is the one doing the wrong thing (in theory), but it is easier to fix the issue here upstream.

https://github.com/anyscale/docs/blob/master/sync_oa_templates.py#L127